packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js

```diff
@@ -0,0 +1,71 @@
+import { settings } from 'carbon-components';
+const { prefix } = settings;
+
+const _htmlTagRegex = /<.*?>/g;
+const _doubleSpaceRegex = /\s{2,}/g;
+const _italicRegex = /[_*](.*?)[_*]/g;
+const _boldRegex = /[_*]{2}(.*?)[_*]{2}/g;
+
+/**
+ * Removes any html tags from a string and keeps inner text if any
+ *
+ * @param {string} str String to be checked for html tags
+ * @returns {string} String with html tags stripped out
+ */
```

Comment:
Add `@private` so this doesn't show up under jsdoc output.

Reply:
Oh yeah, makes sense! Added.